### PR TITLE
CVE-2018-10841

### PR DESCRIPTION
**Pull request by anemec-rh**

### CODE REVIEW

**Comment by anemec-rh** [comment]
Ack, I just sent a signed email to cve@mitre.org.

File: 2018/10xxx/CVE-2018-10841.json [hunk 1]
```diff
@@ -1,18 +1,65 @@
 {
```
Comment (cve-team) [inline 2018/10xxx/CVE-2018-10841.json]: u'all' is not of type u'array'

Comment (cve-team) [reply 2018/10xxx/CVE-2018-10841.json]: 'version_data' should be an array of objects about the version, not a string. Look at the JSON file for CVE-2018-1112 for an example.

File: 2018/10xxx/CVE-2018-10841.json [hunk 2]
```diff
@@ -1,18 +1,69 @@
...
```
Comment (cve-team) [inline 2018/10xxx/CVE-2018-10841.json]: Expecting : delimiter: line 32 column 41 (char 1173)

File: 2018/10xxx/CVE-2018-10841.json (hunk 1 above)
Comment (anemec-rh) [reply 2018/10xxx/CVE-2018-10841.json]: I have changed the string to an array, could you validate again please?

File: 2018/10xxx/CVE-2018-10841.json (hunk 2 above)
Comment (cve-team) [inline 2018/10xxx/CVE-2018-10841.json]: Expecting : delimiter: line 32 column 41 (char 1173)

**Pull request merged**